modules/oe_whitelabel_list_pages/config/install/core.entity_view_display.node.oe_list_page.full.yml

```diff
@@ -0,0 +1,75 @@
+langcode: en
```

Comment:
To have a consisten approach and be in line with the design we need to implement the same solution using oe_w_content_banner as with the other content types

Reply:
Ok, changed.